src/electron/views/device-connect-view/components/android-setup/prompt-choose-device-step.tsx

```diff
@@ -90,7 +90,10 @@ export class PromptChooseDeviceStep extends React.Component<
             rightFooterButtonProps: {
                 text: 'Next',
                 disabled: this.state.selectedDevice === null,
-                onClick: _ => this.props.deps.androidSetupActionCreator.next(),
+                onClick: _ => {
+                    const selectedDevice = this.state.selectedDevice['metadata'];
```

Comment:
I don't know exactly where the DeviceInfo object is being packed into the metadata field, but it is. Failing to unpack it from the metadata field breaks all downstream logic, since it can't access the device id, the device type, or the device name.